### DISCUSSION TITLE
TypeError: combine() argument 2 must  be datetime.time, not none

### DISCUSSION THREAD

**Discussion by Uri20** [community/community#34368] [Other Feature Feedback, Questions, & Ideas]
I'm just learning how to use python, I'm doing a project to store gps data to a postgres database.
What could be the error in the code?

![20220927_123825.jpg](https://user-images.githubusercontent.com/89423710/192599650-a9a921d6-9401-4d25-b024-78c818204e5c.jpg)

**Comment by airtower-luna** [comment]
The error message tells you that the second argument in that method call (`msg.timestamp`) is `None`. That means it doesn't have a value. And naturally "no value" can't be combined with a date to make a `datetime`.

So the next question you need to investigate is *why* it is `None`. I can see that `msg` comes from the `pynmea2.parse(line)` call. Look at that function and its documentation to see why it might return a `None` timestamp.
